cypress/e2e/components/filings/correctionFilings.cy.ts

```diff
@@ -0,0 +1,39 @@
+import { allFilings } from '../../../fixtures/filings/allFilings'
+import { devBCReg } from '../../../fixtures/origins'
+
+context('Correction Filings', () => {
+  it('Non staff don\'t get option', () => {
+    cy.visitBusinessDashFor('businessInfo/ben/active.json', undefined, false, false, undefined, allFilings, false)
+
+    cy.get('[data-cy="header.actions.dropdown"]').should('not.exist')
+  })
+
+  it('Staff should be able to file a correction', () => {
+    cy.visitBusinessDashFor('businessInfo/ben/active.json', undefined, false, false, undefined, allFilings, true)
+    cy.intercept('POST', '**/api/v2/businesses/**/filings', {}).as('correctionFilingsPost')
+
+    cy.get('[data-cy="header.actions.dropdown"]').should('exist')
+    cy.get('[data-cy="header.actions.dropdown"]').first().click()
+    cy.get('[data-cy="header.actions.dropdown"] .fileACorrection').click()
+    cy.get('input[name="correctionType"]').should('exist')
+    cy.get('input[name="correctionType"]').first().click()
+    cy.get('[data-cy="correctionForm.submit"]').should('exist')
+    cy.get('[data-cy="correctionForm.submit"]').click()
+    cy.wait('@correctionFilingsPost')
+    
+    //can't check that it has navigated away without hard checking origin
+    cy.origin(devBCReg, () => {
+      cy.get('body').should('contain', 'BC Registries Account Login')
+    })
+  })
+
+  it('Staff shouldn\'t be able to file a correction against an invalid type', () => {
+    cy.visitBusinessDashFor('businessInfo/ben/active.json', undefined, false, false, undefined, allFilings, true)
+    cy.intercept('POST', '**/api/v2/businesses/**/filings', {}).as('correctionFilingsPost')
+
+    cy.get('[data-cy="header.actions.dropdown"]').should('exist')
```

Comment:
Since this unit test's setup is the same as the one above, can you please make it clear why this test gets a different result than the test above?

Reply:
Because it's picking a different filing, one that is disabled, or not supported for corrections filing

Reply:
Where does it pick a different filing?

Reply:
line 35 .eq(2) the 0 and 1 filings are correctionable

Reply:
OK, I get it.

How about a comment before line 16 saying that the first filing is correctable? And a comment before line 35 saying that filing[2] isn't correctable? Thanks.

Reply:
Oops! I was looking at older code. Thanks for this comment!!!

Resolving.

Reply:
I don't have permission in this repo to resolve my own conversations. Can you please give me Write permission, or just go ahead and resolve this convo? Thanks.